app/distros/openatrium.js

```diff
@@ -64,13 +65,14 @@ function init() {
     );
 
     // The Core Makefile is managed by Atrium and varies by release.
-    var filename = 'drupal-org-core.make?id=' + options.drupalDistroRelease;
-    request('http://cgit.drupalcode.org/openatrium/plain/' + filename,
+    var url = 'http://cgit.drupalcode.org/' + options.drupalDistro.id
+      + '/plain/drupal-org-core.make?id=' + options.drupalDistroRelease;
+    request(url,
       function (error, response, body) {
         if (!error && response.statusCode == 200 && body.length) {
           yo.fs.write(yo.destinationPath('src/drupal-org-core.make'), body);
-          done();
         }
+        done();
       }
     );
   }
```

Comment:
Should this section of changes to openatrium.js be a separate simple pull request since it appears to just fix some issues with how version numbers are handled that might be needed beyond the Lightning support?